README.md

```diff
@@ -3,4 +3,4 @@ The calculation to generate a smart building rating
 
 ## Setup
 - Run `make setup` which will run all the required commands
-- Run `pipenv run python smart-building-rating-calculator/main.py`
\ No newline at end of file
+- Run `pipenv run python smart-building-rating-calculator/main.py`
```

Comment:
Add some end-to-end examples of how to run the function, especially if this repo is to be made public!

Reply:
Done.